.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        os: [Ubuntu, Windows, macOS]
+        php: [7.2, 7.3]
```

Comment:
Since Confomo is a SaaS, we don't need to matrix it; as long as it runs on the version of PHP that's running on our server (and Ubuntu) that's good enough.